src/shared-components/button/index.tsx

```diff
@@ -130,7 +133,7 @@ Button.propTypes = {
   isFullWidth: PropTypes.bool,
   isLoading: PropTypes.bool,
   loading: isLoadingPropFunction,
-  onClick: PropTypes.func.isRequired,
+  onClick: PropTypes.func,
```

Comment:
Some of our codes uses buttons with `type="submit"`, and then attaches an `onSubmit` handler to the form component. As such, I was incorrect to suggest we should make this required. Sorry!